confgenerator/logging_processors.go

```diff
@@ -135,6 +135,7 @@ type LoggingProcessorParseRegex struct {
 	ConfigComponent `yaml:",inline"`
 	ParserShared    `yaml:",inline"`
 	Field           string `yaml:"field,omitempty"`
+	PreserveKey     bool   `yaml:"preserve_key,omitempty"`
```

Comment:
Was it intentional to include this as an exposed config in the YAML? If so, are we going to leave it undocumented for now?

Reply:
Changed in: [8145132](https://github.com/GoogleCloudPlatform/ops-agent/pull/1040/commits/81451323dc4ba31fa2296dcc8c76b75838800967)